spec.html

```diff
@@ -19344,7 +19253,7 @@ <h1>Runtime Semantics: Evaluation</h1>
           1. If IsConstructor(_func_) is *false*, throw a *TypeError* exception.
           1. Let _result_ be ? Construct(_func_, _argList_, _newTarget_).
           1. Let _thisER_ be GetThisEnvironment().
-          1. Perform ? _thisER_.BindThisValue(_result_).
```

Comment:
Similarly as the GetSuperBase comment, it'd be nice to have an assertion somewhere that `_thisER_` is a function environment record.